kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -101,6 +102,8 @@
   public static final Long DEFAULT_WEBSOCKET_TIMEOUT = 5 * 1000L;
   public static final Long DEFAULT_WEBSOCKET_PING_INTERVAL = 1 * 1000L;
 
+  public static final Integer DEFAULT_MAX_CONCURRENT_REQUESTS_PER_HOST = 10;
```

Comment:
I guess in this case all requests are usually going to the same host, so in this case we're really setting the max concurrent requests.